resources/autoscaling_groups.go

```diff
@@ -0,0 +1,806 @@
+				Resolver:    schema.PathResolver("VPCZoneIdentifier"),
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "aws_autoscaling_group_instances",
+				Description: "Describes an EC2 instance.",
+				Resolver:    fetchAutoscalingGroupInstances,
+				Columns: []schema.Column{
+					{
+						Name:        "group_cq_id",
+						Description: "Unique CloudQuery ID of aws_autoscaling_groups table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "availability_zone",
+						Description: "The Availability Zone in which the instance is running.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "health_status",
+						Description: "The last reported health status of the instance",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "id",
+						Description: "The ID of the instance.  This member is required.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("InstanceId"),
+					},
+					{
+						Name:        "lifecycle_state",
+						Description: "A description of the current lifecycle state",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "protected_from_scale_in",
+						Description: "Indicates whether the instance is protected from termination by Amazon EC2 Auto Scaling when scaling in.  This member is required.",
+						Type:        schema.TypeBool,
+					},
+					{
+						Name:        "type",
+						Description: "The instance type of the EC2 instance.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("InstanceType"),
+					},
+					{
+						Name:        "launch_configuration_name",
+						Description: "The launch configuration associated with the instance.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "launch_template_id",
+						Description: "The ID of the launch template",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("LaunchTemplate.LaunchTemplateId"),
+					},
+					{
+						Name:        "launch_template_name",
+						Description: "The name of the launch template",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("LaunchTemplate.LaunchTemplateName"),
+					},
+					{
+						Name:        "launch_template_version",
+						Description: "The version number, $Latest, or $Default",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("LaunchTemplate.Version"),
+					},
+					{
+						Name:        "weighted_capacity",
+						Description: "The number of capacity units contributed by the instance based on its instance type",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:        "aws_autoscaling_group_tags",
+				Description: "Describes a tag for an Auto Scaling group.",
+				Resolver:    fetchAutoscalingGroupTags,
+				Columns: []schema.Column{
```

Comment:
Shouldn't this be json ?

Reply:
it has more than 2 fields so i have decided to store it in separate table. @roneli what do you think about it?

Reply:
sure - let's keep as table.